spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+version = '0.0.1-SNAPSHOT'
+
+description = 'Spring Integration Hazelcast Support'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context', version:'4.1.5.RELEASE'
+    compile group: 'org.springframework.integration', name: 'spring-integration-core', version:'4.1.2.RELEASE'
+    compile group: 'com.hazelcast', name: 'hazelcast', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-spring', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-client', version:'3.4.1'
+    compile group: 'junit', name: 'junit', version:'4.12'
+    testCompile group: 'org.springframework', name: 'spring-test', version:'4.1.5.RELEASE'
+}
```

Comment:
This `build.gradle` looks bad for me.
Please, take a look to the https://github.com/spring-projects/spring-integration-java-dsl how to organize `build` stuff: `publish-maven.gradle`, `settings.gradle`, appropriate plugins for the `build.gradle` etc..